runtime/crab/src/pallets/staking.rs

```diff
@@ -70,8 +72,20 @@ impl darwinia_staking::Stake for KtonStaking {
 	}
 }
 
-frame_support::parameter_types! {
-	pub const PayoutFraction: sp_runtime::Perbill = sp_runtime::Perbill::from_percent(40);
+pub enum OnCrabSessionEnd {}
+impl darwinia_staking::OnSessionEnd<Runtime> for OnCrabSessionEnd {
```

Comment:
`fn inflate()` is not implemented for OnCrabSessionEnd, so it will use default implementation?